src/main/webapp/definitions/alfresco-core.yaml

```
@@ -3174,6 +3184,215 @@ paths:
+      responses:
+        '200':
+          description: Successful response
+        '400':
+          description: |
+            Invalid parameter: nodeId is not a valid format, or is not a file
+                               property name of content is not valid
+            Content's storage state does not allow archive.
+            Invalid archive paramters.
+        '401':
+          description: Authentication failed
+        '403':
+          description: Not authorized
+        '409':
+          description: Content already archived
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/storage-info/{contentPropName}/archive-restore':
+    post:
+      x-alfresco-since: "7.2.0"
+      tags:
+        - storage-info
+      summary: Request to restore given content from archive
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.2.0 and newer versions.
+        It also requires at least one specific implementation of underlying functionality in Cloud Connector(s).
+
+        Request to restore given content from acrhive.
+      operationId: requestRestoreContentFromArchive
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/contentPropNameParam'
+        - name: restoreArchivedContentRequest
+          in: body
+          description: |
+            Restore content from archive request parameters.
+            At the moment there is one parameter being supported which is restore priority.
+            'High' restore priority translates to 'Expedited' Glacier restore tier in AWS S3 and 'High' rehydrate priority in Azure Blob.
+            'Standard' restore priority translates to 'Standard' Glacier restore tier in AWS S3 and 'Standard' rehydrate priority in Azure Blob.
+            Body is not mandatory.
+            High restore priority request body example:
+            ```JSON
+            {
+              "restorePriority": "High"
+            }
+            ```
+            Standard restore priority request body example:
+            ```JSON
+            {
+              "restorePriority": "Standard"
+            }
+            ```
+          required: false
+          schema:
+            $ref: '#/definitions/RestoreArchivedContentRequest'
+      responses:
+        '202':
+          description: Successful response (request accepted)
+        '400':
+          description: |
+            Invalid parameter: nodeId is not a valid format, or is not a file
+                                           property name of content is not valid
+            Content's storage state does not allow restore.
+            Invalid restore paramters.
+        '401':
+          description: Authentication failed
+        '403':
+          description: Not authorized
+        '409':
+          description: Content already restored
```

can be add /restore is in progress?
```suggestion
          description: Content already restored/ Restoration is already in progress
```